app.rb

```diff
@@ -7,6 +7,10 @@
 require 'ifsc'
 require './metrics'
 require 'secure_headers'
+require 'daru'
+
+#load the dataframe on server start
+$df = Daru::DataFrame.from_csv("IFSC.csv")
```

Comment:
Any chance this can be loaded from JSON? There's a lot of effort that goes into keeping the image size minimal, and this increases doubles repo size as well.

Reply:
Is it possible to dump the dataframe to disk? (Like how we do with the redis search). That way the JSON files aren't committed to disc, but only the compressed redis database is.

We could do something similar with keeping the CSV during build, but not for runtime

Reply:
@captn3m0 Dumping the dataframe to disk would result in the same size as the csv as it is the most compact version right? Maybe downloading the csv during runtime from the ifsc repo and deleting it after loading should work?

Reply:
@captn3m0 
Another way to obtain this dataframe would be to

1. Fetch all the keys from the redis database
2. Get the value for each key
3. Transform this data and initialize it into a dataframe

I have written a small script which can do this, but the loading into a dataframe takes about 3 mins and will need to be done everytime the server starts

Reply:
Switching to Redis sounds good 👍🏽